internal/backend/db/dbgorm/envs.go

```diff
@@ -39,9 +39,12 @@ func (db *gormdb) CreateEnv(ctx context.Context, env sdktypes.Env) error {
 	return translateError(db.createEnv(ctx, e))
 }
 
+func (db *gormdb) deleteEnvs(ctx context.Context, ids []string) error {
+	return db.db.WithContext(ctx).Where("env_id IN ?", ids).Delete(&scheme.Env{}).Error
+}
+
 func (db *gormdb) deleteEnv(ctx context.Context, envID string) error {
-	e := scheme.Env{EnvID: envID}
-	return db.db.WithContext(ctx).Delete(&e).Error
+	return db.db.WithContext(ctx).Delete(&scheme.Env{EnvID: envID}).Error
```

Comment:
do you think calling deleteEnvs with one id in the list is a valid option ?

Reply:
actually I think I will remove deleteEnvs :) 
it's a not simple decision. But deleting envs one by one could enforce foreign keys while deleting bunch of them cannot. Will recheck it more and will discuss with you

Reply:
WDYM ?
foreign key constraint should be validated even if you delete the table it self

Reply:
rewritten as u suggested - via deleteEnvs